check-pr-status.mjs

```diff
@@ -7,22 +7,23 @@ import { setTimeout } from 'timers/promises';
 const tries = 10;
 const retryDelay = 30000;
 
-await setTimeout(retryDelay);
+export default async function(github, owner, repository, pull_number) {
+  const repo = repository.replace(new RegExp(`${owner}/`), '')
```

Comment:
might be enough to do `repository.split('/')`?

Reply:
Can't think of a one-liner for that. I would split, use the first part and join the other parts.

Reply:
Seems illegal, so just splitting should be fine: 
<img width="564" alt="image" src="https://user-images.githubusercontent.com/1404810/157755608-a4c6c0a7-9aa9-4f06-98fe-7253e4b6784d.png">